modules/virt-configuring-configmap-for-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-configuring-configmap-for-obsolete-cpu-models_{context}"]
+= Configuring a ConfigMap for obsolete CPU models
+
+Use this procedure to configure a ConfigMap for obsolete CPU models.
+
+.Procedure
+
+* Create a ConfigMap object, specifying the obsolete CPU models in the `obsoleteCPUs` array. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cpu-plugin-configmap <1>
+data: <2>
+  cpu-plugin-configmap:
+    obsoleteCPUs: <3>
+      - "486"
+      - "pentium"
+      - "pentium2"
+      - "pentium3"
+      - "pentiumpro"
+    minCPU: "Penryn" <4>
+
```

Comment:
You have an extra/empty line, can you remove?

Reply:
Removed.